src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfOperation = 0;
+
+  const getSum = (a) => {
+    countOfOperation++;
+
+    const add = (b) => {
+      if (countOfOperation <= 3) {
+        return a + b;
+      }
+
+      if (countOfOperation > 3 && countOfOperation % 2 !== 0) {
+        return a + b;
+      }
+
+      return 'Bzzz... Error!';
```

Comment:
```suggestion
      if (countOfOperation > 3 && countOfOperation % 2 === 0) {
        return 'Bzzz... Error!';
      }
      
      return a + b;
```
it can be done easier